src/Menu/Menu.tsx

```diff
@@ -2,19 +2,22 @@ import {defineMessage} from '@formatjs/intl'
 import {createStackNavigator} from '@react-navigation/stack'
 import React from 'react'
 import {useIntl} from 'react-intl'
-import {Linking, ScrollView, TouchableOpacity, View} from 'react-native'
+import {Image, Linking, ScrollView, StyleSheet, TouchableOpacity, View} from 'react-native'
 import {SafeAreaView} from 'react-native-safe-area-context'
 import {useSelector} from 'react-redux'
 
 import {Text} from '../../legacy/components/UiKit'
 import {CONFIG} from '../../legacy/config/config'
 import {tokenBalanceSelector} from '../../legacy/selectors'
+import FaqImage from '../assets/img/icon/shape.png'
```

Comment:
I'd suggest a name more specific, also please sync with design, to get `@2x` `@3x`, the high-resolution images.